openc3/lib/openc3/microservices/reducer_microservice.rb

```diff
@@ -83,12 +84,25 @@ class ReducerMicroservice < Microservice
     HOUR_ENTRY_NSECS = 3600 * 1_000_000_000
     HOUR_FILE_NSECS = 3600 * 24 * 1_000_000_000
     DAY_ENTRY_NSECS = 3600 * 24 * 1_000_000_000
-    DAY_FILE_NSECS = 3600 * 24 * 30 * 1_000_000_000
+    DAY_FILE_NSECS = 3600 * 24 * 5 * 1_000_000_000
```

Comment:
Did you intentionally leave this at 5?

Reply:
Yes, 30 days is way to long to wait for a file.